api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/InventoryServiceClient.java

```diff
@@ -141,6 +161,8 @@ public Flux<InventoryResponseDTO> getAllInventory(){
                 .retrieve()
                 .bodyToFlux(InventoryResponseDTO.class);
     }
+
+     */
```

Comment:
Can you remove the commented-out code brother? Everything looks good other than that great job Matthew.